### DISCUSSION TITLE
Include com FluentMap

### DISCUSSION THREAD

**Discussion by joao-sotero** [balta-io/forum#182] [Tira Dúvidas] [answered]
Estou no curso de EntityFramework na parte do fluentMap, na ultima do modulo 3, fazemos um inserte com relacionamento, mas como faço um select nesse relacionamento?

![image](https://user-images.githubusercontent.com/62224433/209249349-d995f6c6-1df8-45e6-be25-58fef3ebdba0.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @joao-sotero! Não entendi sua dúvida... este select que você mostrou não funcionou?

**Reply by joao-sotero** [reply]
Bom dia, @andrebaltieri! Não funcionou, diz que o Include não existe.
Baixei o repositorio da aula e o erro é o mesmo.

**Reply by andrebaltieri** [reply]
Você chegou a importar o System.Linq e/ou System.Collection.Generics?

**Reply by joao-sotero** [reply]
sim, o System.Linq, mas nem é utilizado.
![image](https://user-images.githubusercontent.com/62224433/209586256-0bc3da41-a72a-4aec-afc7-e6ad5c475799.png)
![image](https://user-images.githubusercontent.com/62224433/209586303-135dcf11-fb21-4a03-8178-2e8b3161a72e.png)

**Reply by joao-sotero** [reply]
Consegui resolver, faltou o using do EF "using Microsoft.EntityFrameworkCore;"

**Reply by andrebaltieri** [reply]
Showww!
